service/src/main/kotlin/io/provenance/onboarding/frameworks/provenance/extensions/ProvenanceExtensions.kt

```diff
@@ -0,0 +1,35 @@
+package io.provenance.onboarding.frameworks.provenance.extensions
+
+import cosmos.auth.v1beta1.Auth
+import cosmos.base.abci.v1beta1.Abci.TxResponse
+import cosmos.base.tendermint.v1beta1.Query
+import cosmos.tx.v1beta1.ServiceOuterClass.BroadcastTxResponse
+import io.provenance.client.grpc.PbClient
+import io.provenance.onboarding.frameworks.provenance.SingleTx
+import io.provenance.scope.contract.proto.Contracts
+import java.util.concurrent.TimeUnit
+import io.provenance.client.protobuf.extensions.getBaseAccount
+
+fun BroadcastTxResponse.isError() = txResponse.isError()
+
+fun TxResponse.isError() = code > 0
+
+fun TxResponse.isSuccess() = !isError() && height > 0
+
+fun TxResponse.getError(): String =
+    logsList.filter { it.log.isNotBlank() }.takeIf { it.isNotEmpty() }?.joinToString("; ") { it.log } ?: rawLog
+
+fun SingleTx.getErrorResult() =
+    this.value.envelopeState.result.contract.considerationsList.firstOrNull { it.result.result == Contracts.ExecutionResult.Result.FAIL }
+
+fun cosmos.tx.v1beta1.ServiceGrpc.ServiceBlockingStub.getTx(hash: String) = this.withDeadlineAfter(10, TimeUnit.SECONDS)
+    .getTx(cosmos.tx.v1beta1.ServiceOuterClass.GetTxRequest.newBuilder().setHash(hash).build())
```

Comment:
Nit: Add imports for `cosmos.tx.v1beta1.ServiceGrpc.ServiceBlockingStub` and `cosmos.tx.v1beta1.ServiceOuterClass.GetTxRequest` to make this code more digestable?

Reply:
actually the extension isnt even used :awkward: so i'll just delete the whole thing LOL